src/scripts/main.js

```diff
@@ -354,7 +354,23 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const calculateAge = (born, died) => died - born;
+const calculateCentury = (year) => Math.ceil(year / 100);
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach((person) => {
+  const row = document.createElement('tr');
+
+  const gender = person.sex === 'm' ? 'Male' : 'Female';
+
+  row.innerHTML = `
+    <td>${person.name}</td>
+    <td>${gender}</td>
+    <td>${person.born}</td>
+    <td>${person.died}</td>
+    <td>${calculateAge(person.born, person.died)}</td>
+    <td>${calculateCentury(person.born)}</td>
+  `;
+
+  table.appendChild(row);
```

Comment:
Appending the row to the table is correctly implemented. Ensure that the table structure in the HTML is set up to accommodate these rows, including any necessary headers.